arm-apimanagement/2016-10-10/swagger/apimreports.json

```diff
@@ -0,0 +1,266 @@
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Lists a collection of Report record.",
+            "schema": {
+              "$ref": "#/definitions/ReportCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/ReportRecordContract"
+      }
+    }
+  },
+  "definitions": {    
+		"ReportCollection": {
+		  "properties": {
+			"value": {
+			  "type": "array",
+			  "items": {
+				"$ref": "#/definitions/ReportRecordContract"
+			  },
+			  "description": "Page values."
+			},
+			"count": {
+			  "type": "integer",
+			  "format": "int64",
+			  "description": "Total record count number across all pages."
+			},
+			"nextLink": {
+			  "type": "string",
+			  "description": "Next page link if any."
+			}
+		  },
+		  "description": "Paged Report records list representation."
+		},
+		"ReportRecordContract": {
+		  "properties": {
+			"name": {
+			  "type": "string",
+			  "description": "Name depending on report endpoint specifies product, API, operation or developer name."
+			},
+			"timestamp": {
+			  "type": "string",
+			  "format": "date-time",
+			  "description": "Start of aggregation period. The date conforms to the following format: `yyyy-MM-ddTHH:mm:ssZ` as specified by the ISO 8601 standard.\n"
+			},
+			"interval": {
+			  "type": "integer",
+			  "format": "int64",
+			  "description": "Length of agregation period."
+			},
+			"country": {
+			  "type": "string",
+			  "description": "Country to which this record data is related."
+			},
+			"region": {
+			  "type": "string",
+			  "description": "Country region to which this record data is related."
+			},
+			"zip": {
+			  "type": "string",
+			  "description": "Zip code to which this record data is related."
+			},
+			"userId": {
+			  "readOnly": true,
+			  "type": "string",
+			  "description": "User identifier path. /users/{userId}"
+			},
+			"productId": {
+			  "readOnly": true,
+			  "type": "string",
+			  "description": "Product identifier path. /products/{productId}"
+			},
+			"apiId": {
+			  "type": "string",
+			  "description": "API identifier path. /apis/{apiId}"
+			},
+			"operationId": {
+			  "type": "string",
+			  "description": "Operation identifier path. /apis/{apiId}/operations/{operationId}"
+			},
+			"apiRegion": {
+			  "type": "string",
+			  "description": "API region identifier."
+			},
+			"subscriptionId": {
```

Comment:
This conflicts with ```subscriptionId``` path parameter, if you are returning the subscriptionId path, consider naming it as ```subscriptionIdPath```